src/actions/resetPassword/resetNewPasswordAction.test.js

```diff
@@ -0,0 +1,48 @@
+import configureMockStore from "redux-mock-store";
+import { PASSWORD_CHANGE_SUCCESS, PASSWORD_CHANGE_FAILURE } from "./actionTypes";
+import fetchResetPassword from "./resetNewPasswordAction";
+import "babel-polyfill";
+import thunk from "redux-thunk";
+import mockAxios from "axios";
+
+jest.mock('axios');
```

Comment:
- Did you need to make a 2nd  mock of Axios  - on line 6 & 8.
- Do we need to keep  the commented out code

Reply:
Thanks for that feedback. I am working to remove the comments right now plus all the console logs. However, about the two mock Axios lines, yes we do need them both because line 6 simply imports the function that we use to make requests and responses (`mockAxios`) from the `axios` package and line 8 allows jest to mock `axios` requests and responses. Thanks.